### PR TITLE
performance fix in progress 

### PR DESCRIPTION
**Pull request by bshresthap**
performance fix- used Brotli to compress text, added lang on html

### CODE REVIEW

File: client/src/components/ReviewsPhrases.jsx [hunk 1]
```diff
@@ -22,8 +18,11 @@ const ReviewsPhrases = (props) => {
   white-space: nowrap;
   margin: 4px;`;
```
Comment (aleksebastian) [inline client/src/components/ReviewsPhrases.jsx:19]: CSS skills looking good!

File: client/src/components/Reviews.jsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ class Reviews extends React.Component {
         for (var i = 0; i < filteredreviews.length; i++) {
           filteredreviews[i]['readMoreSelected'] = true;
         }
-        console.log('filteredreviews: ', filteredreviews);
+        //console.log('filteredreviews: ', filteredreviews);
 
       })
       .catch(err => console.log('getCustomerReviews Error: ', err))
```
Comment (aleksebastian) [inline client/src/components/Reviews.jsx:65]: Something to consider is what happens on the user side if the code in this catch block is executed.

File: client/src/components/ReviewsPopUp.jsx [hunk 3]
```diff
@@ -70,13 +68,13 @@ const ReviewsPopup = (props) => {
...
+const CloseIconDiv = styled.div`
   text-align: right;
   display: inline-block;
   content: "\00d7"; /* This will render the 'X' */
   `;
 
-  const FlexGallery = styled.div`
+const FlexGallery = styled.div`
   display: flex;
   flex-wrap: wrap;
```
Comment (aleksebastian) [inline client/src/components/ReviewsPopUp.jsx:79]: Nice use of flexbox!

File: client/src/components/chart.jsx [hunk 4]
```diff
@@ -45,7 +46,7 @@ const ChartReview = (props) => {
           }],
           yAxes: [
             {
-              barPercentage: 0.6,
+              // barPercentage: 0.6,
               display: true,
               position: 'left',
               type: 'category',
```
Comment (aleksebastian) [inline client/src/components/chart.jsx:52]: Good job with your chart!

File: package.json [hunk 5]
```diff
@@ -50,6 +49,7 @@
     "@shelf/jest-mongodb": "^1.2.3",
     "babel-jest": "^26.6.3",
     "babel-loader": "^8.0.2",
+    "brotli-webpack-plugin": "^1.1.0",
```
Comment (aleksebastian) [inline package.json:52]: Might need to checkout this webpack plug-in!

File: packageBackup.json [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    "seedData": "node database-mongoose/dataGenerator.js",
+    "test": "jest",
+    "deploy1": "aws s3 cp . s3://s3-fec-reviews --recursive",
+    "deploy2": "aws s3 sync . s3://s3-fec-reviews"
+  },
+  "resolutions": {
+    "styled-components": "^5"
+  },
+  "dependencies": {
+    "axios": "^0.21.1",
```
Comment (aleksebastian) [inline packageBackup.json:33]: I see you commented out your axios imports and added the script for the CDN. Consider removing it from your package.json if you're not using it on your front-end as an import.

**Review by aleksebastian — APPROVED** [review]
Your changes look good. CSS skills on point!

File: packageBackup.json (hunk 6 above)
Comment (bshresthap) [reply packageBackup.json:33]: Good catch! Thanks!

File: client/src/components/Reviews.jsx (hunk 2 above)
Comment (bshresthap) [reply client/src/components/Reviews.jsx:65]: Good point. Made a note of it. Added it to do do list

File: package.json (hunk 5 above)
Comment (bshresthap) [reply package.json:52]: yeah. this helped me with text compression.

**Pull request merged**